### PR TITLE
Add 2020Public version

### PR DESCRIPTION
**Pull request by tofriedl**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [x] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by pilor** [comment]
Approving but please do not skip the "Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently." step in the PR description checklist next time

**Comment by tofriedl** [comment]
I wont.
How can I complete the PR?
I have 1 failing check - Cross Version BreakingChange (Base on stable version): 11 Errors, 0 Warnings failed [Detail<https://github.com/Azure/azure-rest-api-specs/pull/14337/checks?check_run_id=2543467419>]

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-02-02/components_API.json [hunk 1]
```diff
@@ -0,0 +1,766 @@
...
+          "$ref": "#/definitions/PublicNetworkAccessType"
+        },
+        "publicNetworkAccessForQuery": {
+          "description": "The network access type for accessing Application Insights query.",
+          "$ref": "#/definitions/PublicNetworkAccessType"
+        },
+        "IngestionMode": {
+          "type": "string",
+          "description": "Indicates the flow of the ingestion.",
+          "default": "LogAnalytics",
```
Comment (kairu-ms) [inline specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-02-02/components_API.json:594]: The default value changed from "ApplicationInsights" to "LogAnalytics" is it by design?

Comment (kairu-ms) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-02-02/components_API.json:594]: @akning-ms Do we need breaking change review for Cross Version Breaking Change?

**Comment by tofriedl** [comment]
Hey,

Yes it is.

Thanks, Tom.

**Comment by kairu-ms** [comment]
@tofriedl Your PR needs breaking change review.

**Review by kairu-ms — APPROVED** [review]

**Comment by akning-ms** [comment]
One more question is: I saw your latest stable version is 2021-03-08, why you added a new API 2020-02-02, Not sure whether you have someone in your team to coordinate in different feature team for your RP. we recommend you have all your feature APIs available in one API version. ping me in team if need if need more clarify @tofriedl

**Comment by akning-ms** [comment]
it is convert to stable version from 2020-02-02-preview. no any change actually, so merge it.

**Pull request merged**